### DISCUSSION TITLE
如何获取构建时的jobs参数？

### DISCUSSION THREAD

**Discussion by wanzzhehe** [xmake-io/xmake#3676] [Q&A] [answered]
如何在package和target的脚本域中获取jobs参数的值？

**Comment by waruqi** [comment, marked as answer]
https://github.com/xmake-io/xmake/blob/a4a1414b6d81985e4e880c4960ca56016038bee6/xmake/modules/package/tools/autoconf.lua#L471

**Reply by wanzzhehe** [reply]
ok，我知道了

**Reply by wanzzhehe** [reply]
这个 `os.default_njob` 要不要加入到文档里面？毕竟如果是自定义构建规则的话这个应该挺常用的。

**Reply by waruqi** [reply]
可以
